CHANGES.md

```diff
@@ -1,3 +1,7 @@
+Release v0.4.4 (2015-03-31)
+===========================
+* Fix accidential creation of multiple cronjobs when changing the `keep` parameter
+
 Release v0.2.2 (2015-03-30)
```

Comment:
fixed in https://github.com/Brainsware/puppet-tarsnap/commit/f65fa931dcb9a0e5d28bc77a277414c2f57dad39